frontend/src/Pages/AdminManagement.tsx

```diff
@@ -318,72 +281,43 @@ export default function AdminManagement() {
                 ref={addUserModal}
-                type={ModalType.Add}
-                item="Admin"
-                form={
-                    <AddUserForm
-                        onSuccess={onAddUserSuccess}
-                        userRole={UserRole.Admin}
-                    />
-                }
             />
-            <Modal
+            <EditUserModal
+                mutate={mutate}
+                target={targetUser ?? undefined}
                 ref={editUserModal}
-                type={ModalType.Edit}
-                item="Admin"
-                form={
-                    targetUser ? (
-                        <EditUserForm
-                            onSuccess={handleEditUser}
-                            user={targetUser}
-                        />
-                    ) : (
-                        <div>No user defined!</div>
-                    )
-                }
             />
-            <Modal
+            <TextOnlyModal
                 ref={deleteUserModal}
-                type={ModalType.Confirm}
-                item="Delete Admin"
-                form={
-                    <DeleteForm
-                        item="User"
-                        onCancel={handleDeleteUserCancel}
-                        onSuccess={() => void deleteUser()}
-                    />
-                }
-            />
-            <Modal
-                ref={resetUserPasswordModal}
-                type={ModalType.Confirm}
-                item="Reset Password"
-                form={
-                    <ResetPasswordForm
-                        user={targetUser}
-                        onCancel={handleResetPasswordCancel}
-                        onSuccess={handleDisplayTempPassword}
-                    />
+                type={TextModalType.Delete}
+                title={'Delete Admin'}
+                text={
+                    'Are you sure you would like to delete this admin? This action cannot be undone.'
                 }
+                onSubmit={() => deleteUser}
+                onClose={() => handleCancelModal(deleteUserModal)}
             />
-            <Modal
+            <TextOnlyModal
                 ref={showUserPassword}
-                type={ModalType.Show}
-                item={'New Password'}
-                form={
-                    <ShowTempPasswordForm
-                        tempPassword={tempPassword}
-                        userName={
-                            targetUser
-                                ? `${targetUser.name_first} ${targetUser.name_last}`
-                                : undefined
-                        }
-                        onClose={handleShowPasswordClose}
-                    />
-                }
-            />
+                type={TextModalType.Information}
+                title={'New Password'}
+                text={`Copy this password now. If you lose it, you'll need to
+                        regenerate it to get a new one.`}
```

Comment:
Can you update this text to

"Copy your password now. If you lose it, you'll need to generate a new one."